container/container.go

```diff
@@ -188,6 +188,13 @@ func (container *Container) SetupWorkingDirectory() error {
 	if container.Config.WorkingDir == "" {
 		return nil
 	}
+
+	// If can't mount container FS at this point (eg Hyper-V Containers on
+	// Windows) bail out now with no action.
+	if !container.canMountContainerFS() {
```

Comment:
@jhowardmsft Was more refering to the fn name, not the caller.
So for `container.<blah>`, `<blah>` doesn't need to contain `Container`